data/human/news.txt

```diff
@@ -801,6 +872,25 @@ news "geological engineer"
 
 
 
+news "electrical engineer"
+	location
+		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
+	name
+		word
+			"Electrical engineer"
+			"Power networks expert"
+	message
+		word
+			`"`
+		word
+			"Earth has a weird power-distribution network, where the polarity of home outlets changes 50 or 60 times per second. Every other planet just uses a positive and negative polarity on outlets."
+			"Voltage conversion from DC power to DC power works extremely well on every planet except Earth. Earths ancient power-network still uses AC power for distribution."
+			"Most equipment works from 100 DC to 500 DC power, but just check compatibility before connecting your equipment."
```

Comment:
"works from 100 DC to 500 DC" implies that this is some kind of measure of time. Is there a better way to put it?

Reply:
I removed the DC part, this also works just with voltages.